templates/Layout/Page_results_solr.ss

```diff
@@ -23,34 +23,34 @@
                         </a>
                     </h4>
                     <p><% if $Abstract %>$Abstract.XML<% else %>$Content.ContextSummary<% end_if %></p>
-                    <a class="readMoreLink" href="$Link" title="Read more about &quot;{$Title}&quot;">Read more about &quot;{$Title}&quot;...</a>
+                    <a class="readMoreLink" href="$Link" title="Read more about &quot;{$Title}&quot;"><%t SolrResultsPage.ReadMore 'Read more about' %> &quot;{$Title}&quot;...</a>
```

Comment:
the title here isn’t translated - since it’s the same as the content, it doesn’t provide any extra value, maybe you could remove the attribute

Reply:
Maybe I just add translation to title also, as other links uses title attribute?

Reply:
You can, but it provides no value when the title is the same as the content, so may as well be removed. Either way this is an improvement, thank you!